finat/point_set.py

```diff
@@ -34,15 +39,15 @@ def expression(self):
 
 
 class PointSingleton(AbstractPointSet):
-    """A point set representing a single point.
+    """A point set representing a single scalar point.
```

Comment:
I find the term "scalar point" confusing, that sounds like a 1-dimensional point or something.

Reply:
Removed in 8333ae7bcca2fd963d3b952713d66be8b8026749